docs/reference/string/lowerCase.md

```diff
@@ -0,0 +1,30 @@
+# lowerCase
+
+Converts a string to lower case.
+
+Snake case is the naming convention in which each word is written in lowercase and separated by an space ( ) character. For example, `lower_case`.
```

Comment:
This seems to be a description of Snake Case, not LowerCase. It would be better if it were corrected like this. 

```suggestion
Lower case is the naming convention in which each word is written in lowercase and separated by an space ( ) character. For example, `lower case`.
```